web/src/lib/components/shared-components/search-bar/search-bar.svelte

```diff
@@ -17,28 +18,19 @@
   let showFilter = false;
   $: showClearIcon = value.length > 0;
 
-  function onSearch() {
-    let smartSearch = 'true';
-    let searchValue = value;
-
-    if (value.slice(0, 2) == 'm:') {
-      smartSearch = 'false';
-      searchValue = value.slice(2);
-    }
-
-    $savedSearchTerms = $savedSearchTerms.filter((item) => item !== value);
-    saveSearchTerm(value);
+  function onSearch() {}
 
+  const onSearchNew = (payload: SmartSearchDto | MetadataSearchDto) => {
     const parameters = new URLSearchParams({
-      q: searchValue,
-      smart: smartSearch,
-      take: '100',
+      q: JSON.stringify(payload),
```

Comment:
It seems strange to have a full JSON as the value of `q`. Wouldn't it be better for each filter to be a different parameter?

Reply:
This makes easier downstream to grab the whole string as JSON string and decode into javascript object

Reply:
You made the search endpoints POST, so what's the purpose of doing this instead of putting the JSON in the body?